kinto/config/kinto.tpl

```diff
@@ -65,22 +119,67 @@ multiauth.policies = basicauth
 # Set user 'account:admin' as the administrator.
 # kinto.account_write_principals = account:admin
 # kinto.account_read_principals = account:admin
+#
+# Kinto-portier authentication
+# https://github.com/Kinto/kinto-portier
+# multiauth.policies = portier
+# multiauth.policy.portier.use = kinto_portier.authentication.PortierOAuthAuthenticationPolicy
+# kinto.portier.broker_url = https://broker.portier.io
+# kinto.portier.webapp.authorized_domains = *.github.io
+# kinto.portier.cache_ttl_seconds = 300
+# kinto.portier.state.ttl_seconds = 3600
 
+# Notifications
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#notifications
 #
-# Firefox Accounts configuration.
-#   These are working FxA credentials for localhost:8888
-# kinto.includes  = kinto_fxa
-# fxa-oauth.client_id = 61c3f791f740c19a
-# fxa-oauth.client_secret = b13739d8a905315314b09fb7b947aaeb62b47c6a4a5efb00c378fdecacd1e95e
-# fxa-oauth.oauth_uri = https://oauth-stable.dev.lcip.org/v1
-# fxa-oauth.requested_scope = profile kinto
-# fxa-oauth.required_scope = kinto
-# fxa-oauth.relier.enabled = true
-# fxa-oauth.webapp.authorized_domains = *
+# Configuration example:
+# kinto.event_listeners = redis
+# kinto.event_listeners.redis.use = kinto_redis.listeners
+# kinto.event_listeners.redis.url = redis://localhost:6379/0
+# kinto.event_listeners.redis.pool_size = 5
+# kinto.event_listeners.redis.listname = queue
+# kinto.event_listeners.redis.actions = create
+# kinto.event_listeners.redis.resources = bucket collection
```

Comment:
I suggest that you put the production settings just right below this section. They are not used immediately by someone trying the project, so no need to have them on top. 
Plus, the following settings (cross, backoff, end of service) are all production settings too